tensorflow/python/framework/dtypes.py

```diff
@@ -270,6 +293,19 @@ def __hash__(self):
   def size(self):
     return np.dtype(self.as_numpy_dtype).itemsize
 
+# Define data type range of numpy dtype
+dtype_range = {np.bool_: (False, True),
```

Comment:
Define the `dtype_range` for computing the `scale` of dtype, according to [scikit-image dtype range](https://github.com/scikit-image/scikit-image/blob/master/skimage/util/dtype.py#L8)